docs/image-based-upgrade.md

```diff
@@ -149,24 +149,24 @@ LCA will reject the stage transition if it is an invalid transition.
 
 This table presents a comprehensive list of conditions and valid next stages based on the current stage.
 
-|Current Stage|Condition         |Status|Reason        |After Pivot|Valid Next Stages|
-|-------------|------------------|------|--------------|-----------|-----------------|
-|`Idle`       |Idle              |True  |Idle          |N/A        |Prep             |
-|             |                  |False |Aborting      |False      |N/A              |
-|             |                  |False |Finalizing    |True       |N/A              |
-|             |                  |False |AbortFailed   |False      |N/A              |
-|             |                  |False |FinalizeFailed|True       |N/A              |
-|`Prep`       |PrepInProgress    |True  |InProgress    |False      |Idle             |
-|             |PrepCompleted     |False |Failed        |False      |Idle             |
-|             |                  |True  |Completed     |False      |Idle, Upgrade    |
-|`Upgrade`    |UpgradeInProgress |True  |InProgress    |False      |Idle             |
-|             |                  |True  |InProgress    |True       |Rollback         |
-|             |UpgradeCompleted  |False |Failed        |False      |Idle             |
-|             |                  |False |Failed        |True       |Rollback         |
-|             |                  |True  |Completed     |True       |Idle, Rollback   |
-|`Rollback`   |RollbackInProgress|True  |InProgress    |True       |N/A              |
-|             |RollbackCompleted |False |Failed        |True       |N/A              |
-|             |                  |True  |Completed     |True       |Idle             |
+| Current Stage | Condition          | Status | Reason         | After Pivot | Valid Next Stages |
```

Comment:
Nit: Looks like an accidental format or did something change in the state table?

Reply:
yea, nothing actually changed in the state table... was just my IDE complaining about "bad table formatting"